custom_components/tahoma/tahoma_device.py

```diff
@@ -1,14 +1,12 @@
 """Parent class for every TaHoma devices."""
-from datetime import timedelta
 from typing import Any, Optional
 
-from tahoma_api.client import TahomaClient
-from tahoma_api.models import Command, Device
+from tahoma_api.models import Command
 
 from homeassistant.const import ATTR_BATTERY_LEVEL
 from homeassistant.helpers.entity import Entity
-from homeassistant.util import Throttle
 
+from . import TahomaDataUpdateCoordinator
```

Comment:
```suggestion
from .coordinator import TahomaDataUpdateCoordinator
```

Reply:
That's isort that do it. I cannot change it.